README.md

```diff
@@ -34,6 +34,19 @@ And if you need to use a specific version of the package, you can use something
 #load nuget:?package=Chocolatey.Cake.Recipe&prerelease&version=0.1.0-unstable0026
 ```
 
+### Local Development
+
+The easiest way to do local development of this package is to load the latest released version from NuGet, and exclude the cake files you're iterating.
+
+For example, to work with the `sign.cake` file, your `recipe.cake` file might include:
+
+```
+#load nuget:?package`Chocolatey.Cake.Recipe&Version=0.30.1&Exclude=/**/sign.cake
+#load nuget:?path=C:/code/Chocolatey.Cake.Recipe/Chocolatey.Cake.Recipe/Content/sign.cake

```

Comment:
```suggestion
#load local:?path=C:/code/Chocolatey.Cake.Recipe/Chocolatey.Cake.Recipe/Content/sign.cake
```

Without this change, I was getting the following:

![image](https://github.com/user-attachments/assets/3760bfe8-8c62-42b5-b648-58a89f54674b)

Reply:
That would be correct 😂 I followed the Cake docs, and when updating the readme just free-handed it. That'll teach me. Next time I should just copy/paste from the working file.